src/drivers/distance_sensor/vl53l1x/vl53l1x.hpp

```diff
@@ -117,6 +117,15 @@ class VL53L1X : public device::I2C, public I2CSPIDriver<VL53L1X>
 	 */
 	void RunImpl();
 
+	// Distance mode member variable
+	uint16_t distance_mode;
+
+	// Zone index member variable
+	uint8_t zone_index;
+
+	// Zone limit member variable
+	uint8_t zone_limit;
```

Comment:
```suggestion
	uint8_t _zone_limit{0};
```

As a habit it's safer to initialize everything.